nservicebus/azure-storage-queues/configuration.md

```diff
@@ -97,25 +98,7 @@ WARNING: The values of `BatchSize` , `DegreeOfParallelism`, `Concurrency`, [Serv
 
 NOTE: `QueueName` and `QueuePerInstance` are obsoleted. Instead, use bus configuration object to specify the endpoint name and select a scale-out option.
```

Comment:
After reading this piece several times and going through the code changes, I'd drop this paragraph.